tracking/include/pcl/tracking/particle_filter.h

```diff
@@ -1,507 +1,639 @@
 #pragma once
 
-#include <pcl/memory.h>
 #include <pcl/filters/passthrough.h>
 #include <pcl/octree/octree_pointcloud_changedetector.h>
 #include <pcl/tracking/coherence.h>
 #include <pcl/tracking/tracker.h>
 #include <pcl/tracking/tracking.h>
 
+#include <boost/shared_ptr.hpp>
```

Comment:
I thought we already remove all all occurrences of boost::shared_ptr from the code base. Why is this back?